src/org/python/core/RegistryKey.java

```diff
@@ -0,0 +1,297 @@
+package org.python.core;
+
+/**
+ *
+ * Supported registry keys and their usage. 
+ *
+ * Boolean properties are set with the 
+ * String values {@code true} or {@code yes} for true, or {@code false} or 
+ * {@code no} for false.
+ *
+ */
+public class RegistryKey {
+
+    private RegistryKey(){}
+
+    /**
+     *
```

Comment:
Is there a reason why some (i.e. most) comments have a leading blank line and some don't?

Reply:
I'm used to a style where there is a leading blank line. I agree it's better to be consistent. Looking it up now, it seems Oracle doesn't have it.

https://www.oracle.com/technetwork/articles/javase/index-137868.html

Perhaps it used to be a default in Elipse or something? Anyway I will make them consistent and remove the leading blank line, unless there is a strong preference otherwise or some other existing style guide.

Reply:
I find a space above and the line /** is all the blank I want. Thanks for the change.